recipes/pykafka/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pykafka" %}
+{% set version = "2.6.0" %}
+{% set sha256 = "19e4a802b00e28c3203fcfd80ca440b13faecb8e58171e9685a044cf87dae93e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - librdkafka  # [not win]
```

Comment:
Normally for libraries that are linked to, we pick some sort of pinning so the stack remains consistent and things don't break when a new version comes out. Based on the [API/ABI report]( https://abi-laboratory.pro/tracker/timeline/librdkafka/ ), it looks like `librdkafka` breaks on patch versions sometimes. Though `0.9.4` and `0.9.5` seem ok. Given this, it seems reasonable to change this to `librdkafka 0.9.4|0.9.5`. Thoughts on this?

cc @conda-forge/librdkafka

Reply:
:+1: I'm pinned at `0.9.4` here; this works for me.

I'd argue it's important to keep all of the libraries that depend on `librdfakfa` on the same pin so they are interchangable. Is there a non-manual way to do this other than a global `conda-forge` pin?

*  https://github.com/conda-forge/python-confluent-kafka-feedstock
*  https://github.com/conda-forge/kafkacat-feedstock

Reply:
Hrm, at runtime I would expect conda itself to figure this out.  I'm not sure how feasible it is to use pykafka with a different librdkafka from which it was built.